gtrending/cli.py

```diff
@@ -0,0 +1,187 @@
+import argparse
+import json
+
+from typing import Dict, Any
+
+from .fetch import (
+    fetch_repos,
+    fetch_developers,
+    languages_list,
+    spoken_languages_list,
+)
+
+
+def main():
+
+    parser = argparse.ArgumentParser('Gtrending')
+    subparser = parser.add_subparsers(dest='command')
+  
+    # Extract common arguments in parents
+    parent_json = argparse.ArgumentParser(add_help=False)
+    parent_json.add_argument(
+        '--json',
+        '-J',
+        action=argparse.BooleanOptionalAction,
+        default=False,
```

Comment:
Making JSON output off, with a human-readable key-value output by default seems like a good idea! If scripts are to use this maybe they could sed or awk to do some custom filters with it, if they're not using jq. (By the way, we could perhaps have examples with jq filters in the README)